staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/marshal_test.go

```diff
@@ -17,9 +17,14 @@ limitations under the License.
 package v1
 
 import (
+	"encoding/hex"
 	"encoding/json"
 	"reflect"
+	"strings"
 	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	cbor "k8s.io/apimachinery/pkg/runtime/serializer/cbor/direct"
 )
 
 type JSONSchemaPropsOrBoolHolder struct {
```

Comment:
I don't want to suggest blowing up the diff of an already substantial change, otherwise I think I would like to see the "Holder" test types removed in favor of directly testing the MarshalJSON and UnmarshalJSON methods. We're partially exercising behaviors of `encoding/json`, like the handling of omitempty field tags.